src/photos/ducks/albums/index.js

```diff
@@ -118,15 +124,49 @@ const ConnectedAddToAlbumModal = props => (
 
 const ConnectedAlbumPhotos = withRouter(props => (
   <Query query={ALBUM_QUERY} {...props} mutations={ALBUM_MUTATIONS}>
-    {({ data }, { updateAlbum, deleteAlbum, removePhotos }) => (
-      <AlbumPhotos
-        album={data}
-        updateAlbum={updateAlbum}
-        deleteAlbum={deleteAlbum}
-        removePhotos={removePhotos}
-        {...props}
-      />
-    )}
+    {(
+      { data, hasMore, fetchMore, fetchStatus },
+      { updateAlbum, deleteAlbum, removePhotos }
+    ) => {
+      if (fetchStatus === 'loaded') {
+        return (
+          <Query query={ALBUM_GET_ONE} {...props} as="toto">
```

Comment:
```suggestion
          <Query query={ALBUM_GET_ONE} {...props}>
```

Reply:
We have to do this double query to fetch the album's name + the photos, correct?

Reply:
Yep. I think we have something like `queryConnect` to run in parallel some queries, but it's something overkill for our use case here. Nop?